packages/@aws-cdk/aws-codedeploy/lib/lambda/deployment-group.ts

```diff
@@ -136,24 +136,15 @@ export class LambdaDeploymentGroup extends cdk.Construct implements ILambdaDeplo
     this.application = props.application || new LambdaApplication(this, 'Application');
     this.alarms = props.alarms || [];
 
-    let serviceRole: iam.Role | undefined = props.role;
-    if (serviceRole) {
-      if (serviceRole.assumeRolePolicy) {
-        serviceRole.assumeRolePolicy.addStatement(new iam.PolicyStatement()
```

Comment:
This part of the behavior is not retained by the new code. That constitutes a breaking change.

Either it needs to be restored (if `props.role` is set, then make the role assumable by `code deploy.amazonaws.com`), or the breaking change needs to be mentioned.

Reply:
You are right.

I removed it because this doesn't match our behavior anywhere else where we do BOYR, but it does constitute a breaking change. And the build is probably failing because there's a test asserting that we DO do this :).